setup.cfg

```diff
@@ -47,7 +47,8 @@ doc =
 test =
     katsdpsigproc[CUDA]
     async-solipsism
-    pytest
+    async-timeout
```

Comment:
I think this might be a merge issue - async-timeout was removed as part of the Python 3.12 work, and it's snuck back in here.

Reply:
Yeah I was wondering where that came from. I put it down to me manually removing packages before running `pip-compile` - which, in hindsight, shouldn't have affected `setup.cfg`.